### DISCUSSION TITLE
regarding the issue of dragging within a group

### DISCUSSION THREAD

**Discussion by hsfzmsn** [jgraph/drawio#4172] [General]
after merging the groups, why can I still click on the internal components within the group and drag them out? I don't want to do this because I want them to be treated as a whole. How can I achieve this? Thanks

**Comment by babukiika** [comment]
I do not see the rectangular shape from the left of the draw.io so I drag it

**Comment by hsfzmsn** [comment]
How do I put the SVG format tools I made in the General toolbar?

**Comment by hsfzmsn** [comment]
It’s my customized tool
